bety_brapi/controllers_impl/StudiesController_impl.py

```diff
@@ -40,3 +40,106 @@ def seasons_get(year=None, pageSize=None, page=None):
         })
 
     return helper.create_result({"data": data}, count, pageSize, page)
+
+
+def studies_study_db_id_get(studyDbId):
+
+    params = list()
+
+    query = "SELECT experiments.id as studyDbId, " \
+            "   experiments.name as studyName, " \
+            "   experiments.start_date as startDate, " \
+            "   experiments.end_date as endDate, " \
+            "   experiments.description as studyDescription, " \
+            "   experiments_sites.site_id as location_name, " \
+            "   sites.sitename as location_abbreviation " \
+            "FROM experiments, experiments_sites, sites " \
+            "WHERE experiments.id = experiments_sites.experiment_id " \
+            "AND sites.id = experiments_sites.site_id " \
+            "AND experiments.id = " + studyDbId
```

Comment:
is this safe from a sql injection attack? is there a type check anywhere?

Reply:
@tcnichol for example on how to use parameterized queries see :
https://github.com/terraref/brapi/blob/master/bety_brapi/controllers_impl/PhenotypesController_impl.py#L33 and https://github.com/terraref/brapi/blob/master/bety_brapi/controllers_impl/PhenotypesController_impl.py#L58

Reply:
I will redo this part based on the other sql queries.

Reply:
I think I did this right, but not sure. Rob, can you check?

Reply:
If I understand correctly I think that given the current lines 58-59:

```python
            "AND sites.id = experiments_sites.site_id " \
            "AND experiments.id = " + studyDbId
```

Should be replaced by:

```python
            "AND sites.id = experiments_sites.site_id "
    if studyDbId:
        query += " and es.experiment_id = %s "
        params.append(studyDbId)
```

Reply:
sure thing. will change the line